IEnvoyProxy/go.mod

```diff
@@ -11,15 +11,12 @@ replace (
 )
 
 require (
+	git.torproject.org/pluggable-transports/snowflake.git/v2 v2.0.0-00010101000000-000000000000
 	github.com/lucas-clemente/quic-go v0.27.2 // indirect
-	github.com/onsi/ginkgo v1.16.5 // indirect
-	github.com/onsi/gomega v1.17.0 // indirect
-	github.com/pion/dtls/v2 v2.1.5 // indirect
-	github.com/pion/sctp v1.8.2 // indirect
-	github.com/pion/transport v0.13.0 // indirect
```

Comment:
what were these dependencies from?

Reply:
probably DNSTT? I hasn't run `go mod tidy` in a while